index.js

```diff
@@ -17,7 +16,27 @@ const manageRecipes = async () => {
 
     // Before adding any recipes to the database, let's remove all existing ones
     await Recipe.deleteMany();
-
+    await Recipe.create({
+      title: "Torrada",
+      level: "Easy Peasy",
+      ingredients: ["Bread", "Ham", "Cheese"],
+      cuisine: "unknown",
+      dishType: "breakfast",
+      image:
+        "https://media.istockphoto.com/id/185235465/photo/white-toast.jpg?s=612x612&w=0&k=20&c=ZIMknnJMwf9PANFFWeR78s_d72lDLIBi1ZSeunfnE34=",
+      duration: 2,
+      creator: "unknown",
+    });
```

Comment:
Here they ask you to console.log the title of the recipe that you created. One way of doing this is storing the Recipe.create in a variable:

```
const oneRecipe = await Recipe.create({
      title: "Torrada",
      level: "Easy Peasy",
      ingredients: ["Bread", "Ham", "Cheese"],
      cuisine: "unknown",
      dishType: "breakfast",
      image:
        "https://media.istockphoto.com/id/185235465/photo/white-toast.jpg?s=612x612&w=0&k=20&c=ZIMknnJMwf9PANFFWeR78s_d72lDLIBi1ZSeunfnE34=",
      duration: 2,
      creator: "unknown",
    });
    console.log(`New recipe added: ${oneRecipe.title}`);
```